.npmrc

```diff
@@ -1 +1 @@
-registry=https://art-bobcat.autodesk.com/artifactory/api/npm/autodesk-npm-virtual/
\ No newline at end of file
+registry=https://registry.npmjs.org
```

Comment:
so is there another .npmrc file for the build that points to art-bobcat?

Reply:
That file will be switched during CI/CD process as requested on the Jira task